speech-to-text/recognize_stream.js

```diff
@@ -217,7 +217,7 @@ RecognizeStream.prototype.initialize = function() {
   this.initialized = true;
 };
 
-RecognizeStream.prototype._read = function(/* size*/) {
+RecognizeStream.prototype._read = function() /* size*/ {
```

Comment:
This change should be reverted (if the linter did it, then you may have an outdated version of `pretier` installed - run `npm update` and then run `npm run autofix`)